frontend/ui/widget/textboxwidget.lua

```diff
@@ -835,11 +835,16 @@ function TextBoxWidget:_renderText(start_row_idx, end_row_idx)
     h = h + self.line_glyph_extra_height
     if self._bb then self._bb:free() end
     local bbtype = nil
-    if self.line_num_to_image and self.line_num_to_image[start_row_idx] then
+    local colorful_bg_pen = not Blitbuffer.isColor8(self.bgcolor)
```

Comment:
`_pen` ? as in "a pen for writing" ? but we're filling, so dunno if "pen" is the right word or even needed here.

Reply:
Hahahaha🤣

Reply:
Yep, pencil, there was a vague idea of doing the same for fg, where pen sort of makes more sense ;p.